react/components/sellerOnboardingModal.tsx

```diff
@@ -63,6 +63,7 @@ const SellerOnboardingModal: FC<any> = ({ seller, dispatch, disabled }) => {
         },
       })
     } catch (error) {
+      // eslint-disable-next-line no-console
```

Comment:
Using console.error instead of console.log should allow you to keep this eslint rule enabled